### PR TITLE
ES2015ify and support Node.js >=4

### PR DESCRIPTION
**Pull request by gillstrom**
Did I miss anything @kevva?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,24 +1,25 @@
 'use strict';
-var linuxBattery = require('linux-battery');
-var osxBattery = require('osx-battery');
-var toDecimal = require('to-decimal');
+const execFile = require('get-exec-file');
```
Comment (kevva) [inline index.js]: Maybe use `execa` instead?

**Comment by kevva** [comment]
Set the following in your `package.json` to enforce ES2015 syntax too:

```
"xo": {
  "esnext": true
}
```

File: index.js (hunk 1 above)
Comment (gillstrom) [reply index.js]: Haha of course.. for some reason I thought WMIC was a file.

**Comment by gillstrom** [comment]
Fixedddd, nice new profile photo btw

**Comment by kevva** [comment]
> nice new profile photo btw

+1

**Comment by sindresorhus** [comment]
> nice new profile photo btw

👍

**Pull request merged**